.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+    // Verwendet IntelliSense zum Ermitteln möglicher Attribute.
```

Comment:
Does this have to be in German? I prefer english comments.

Reply:
Der Kommentar stammt nicht vom Visual STudio Code. Aber das File sollte auch gar nicht eingecheckt werden. War ein Versehen.